sdk/monitor/query/azlogs/build.go

```diff
@@ -1,8 +1,6 @@
-//go:build go1.18
-// +build go1.18
-
-//go:generate autorest ./autorest.md
-//go:generate gofmt -w .
+//go:generate tsp-client update --local-spec-repo /home/grace/code/azure-rest-api-specs/specification/monitor/Monitor.Query.Logs
```

Comment:
Does `--local-spec-repo` need to be removed?

Reply:
Thanks! Fixed in [c8d3d56](https://github.com/Azure/azure-sdk-for-go/pull/24955/commits/c8d3d56885e404e515bb9d2c9c9f2e32cf6ea03b)